src/App.css

```diff
@@ -1,38 +0,0 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
```

Comment:
Why is this file empty?

Reply:
@cariowix 

it shouldn't even be empty in this pr....
removing this app and replacing the landing page should be in the next pr

Reply:
ive restored it to wokring state, and it will be changeed with your next pr.